src/Controller/ScheduledTaskController.php

```diff
@@ -77,6 +77,25 @@ public function scheduleTask(Request $request, string $id, Context $context): Js
         return new JsonResponse(null, Response::HTTP_NO_CONTENT);
     }
 
+    #[Route(path: '/scheduled-task/deactivate/{id}', name: 'api.frosh.tools.scheduled.task.deactivate', methods: ['POST'])]
+    public function deactivateTask(string $id, Context $context): JsonResponse
```

Comment:
we need a way to activate it again? 🤔

Reply:
Activating is done through scheduling the task, via endpoint `api.frosh.tools.scheduled.task.schedule`